pydicom/tests/test_uid.py

```diff
@@ -112,25 +40,180 @@ def testGenerateUID(self):
             self.assertEqual(uid[:len(valid_prefix)], valid_prefix)
             self.assertTrue(len(uid) <= 64)
 
-    def testIsValid(self):
+
+class TestUID(unittest.TestCase):
```

Comment:
Can you try to write the tests using pytests fixtures? that would simplify the tests. now that we are testing with pytest.